<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptBatcher</class>
 <widget class="QGroupBox" name="ScriptBatcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="title">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="scriptsFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="scriptsUpButtonsLayout">
        <item>
         <widget class="QLabel" name="scriptsFrameHeader">
          <property name="text">
           <string>Scripts</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="scriptsUpButtonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="scriptsAddButton">
          <property name="text">
           <string>Add Scripts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scriptsRemoveButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="scriptsList">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
